triage/experiments/validate.py

```diff
@@ -111,55 +143,88 @@ def run(self, state_config):
+                If a table_name is given in state_config,
+                at least one state filter must be present''')
+        else:
+            logging.warning('No table_name found in state_config.' +
+                            'The provided events table will be used, which ' +
+                            'may result in unnecessarily large matrices')
 
 
 class FeatureGroupDefinitionValidator(Validator):
     def run(self, feature_group_definition, feature_aggregation_config):
         if not isinstance(feature_group_definition, dict):
-            raise ValueError('Feature Group Definition must be a dictionary')
+            raise ValueError('''Section: feature_group_definition -
+            feature_group_definition must be a dictionary''')
 
         available_subsetters = architect.feature_group_creator.FeatureGroupCreator.subsetters
         for subsetter_name, value in feature_group_definition.items():
             if subsetter_name not in available_subsetters:
-                raise ValueError('''Unknown feature_group_definition key {} received.
+                raise ValueError('''Section: feature_group_definition -
+                Unknown feature_group_definition key {} received.
                 Available keys are {}'''.format(subsetter_name, available_subsetters))
             if not hasattr(value, '__iter__') or isinstance(value, (str, bytes)):
-                raise ValueError('Each value in FeatureGroupCreator must be iterable and not a string')
+                raise ValueError('''Section: feature_group_definition -
+                feature_group_definition value for {}, {}
+                should be a list'''.format(subsetter_name, value))
 
         if 'prefix' in feature_group_definition:
-            available_prefixes = [
+            available_prefixes = {
                 aggregation['prefix']
                 for aggregation in feature_aggregation_config
-            ]
-            for prefix in feature_group_definition['prefix']:
-                if prefix not in available_prefixes:
-                    raise ValueError('''Aggregation prefix of '{}' was given as a
-                    feature group, but no such prefix exists in the available
-                    feature aggregations. The available feature aggregations are {}
-                    '''.format(prefix, available_prefixes))
+            }
+            bad_prefixes = set(feature_group_definition['prefix']) - available_prefixes
+            if bad_prefixes:
+                raise ValueError('''Section: feature_group_definition -
+                The following given feature group prefixes: '{}'
+                are invalid. Available prefixes from this experiment's feature
+                aggregations are: '{}'
+                '''.format(bad_prefixes, available_prefixes))
+
+        if 'tables' in feature_group_definition:
+            available_tables = {
+                aggregation['prefix'] + '_aggregation'
```

Comment:
oh, actually with the imputation branch merged, this should be `aggregation['prefix'] + '_aggregation_imputed'`

that said, looks like we actually need to fix the example of this in the example experiment config [here](https://github.com/dssg/triage/blob/master/example_experiment_config.yaml#L175) as well if you don't mind doing that along with this PR.